js/ui/viewSelector.js

```diff
@@ -327,6 +419,7 @@ const ViewsDisplay = new Lang.Class({
     _init: function() {
         this._enterSearchTimeoutId = 0;
         this._localSearchMetricTimeoutId = 0;
+        this.discoveryFeed = maybeCreateDiscoveryFeed();
```

Comment:
I don't think there's a reason to make this "public", so probably should be `this._discoveryFeed`